### ISSUE TITLE
Admin pages should be invisible to non-admins

### ISSUE THREAD

**Issue by mrkvon** [Trustroots/trustroots#1313]
#### What I see

I'm not an admin. When I go to https://trustroots.org/admin, I can see the admin interface.

![image](https://user-images.githubusercontent.com/7449720/77121965-baa24480-6a3c-11ea-898d-0401ada880c6.png)

#### What I expect

Instead of the admin interface, I'd like to see an info like `Sorry, you're not an administrator.`

#### Additional info

API requests fail with 403, so I can't get access to any sensitive data. Therefore, this is not a security issue. Just a UX imperfection.

**Comment by simison** [comment]
Yep, I saw there's some console log warning coming from the router. I suspect it's because we now hide `user.roles` from the frontend. We should change it so that we send those still for admins.
